scripts/release.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+set -e
+
+REPOSITORY="${REPOSITORY:-git@github.com:etcd-io/etcd-operator.git}"
+REGISTRY="${REGISTRY:-gcr.io/etcd-development/etcd-operator}"
+CONTAINER_TOOL="${CONTAINER_TOOL:-docker}"
```

Comment:
Out of curiosity have you tested with `podman`?

Reply:
Would [ko-build/ko](https://github.com/ko-build/ko) also be a good option?

Reply:
we can continue to enhance the release script in future if needed.

Reply:
Podman doesn't work because of the built-in cross architecture build from the scaffolded `Makefile`. I agree; we can iterate and enhance it.